lib/vdsm/tool/configurators/multipath.py

```diff
@@ -109,3 +106,25 @@ def isconfigured():
 
     sys.stdout.write("multipath requires configuration\n")
     return NO
+
+
+def isconfigured():
+    """
+    Check the multipath daemon configuration.
+    """
+    conf_metadata_ok = _check_mpath_metadata()
+    try:
+        mpathconf.check_local_config()
+    except exception.InvalidConfiguration as e:
+        ufn_enabled_sections = e.context["sections"]
```

Comment:
Using the context works, but it will be better to handle error formatting in the exception class.